core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/commands.scala

```diff
@@ -87,3 +87,14 @@ case class WriteStreamOperationStart(
     foreach_batch: Option[StreamingForeachFunction])
     extends LeafNode
     with Command
+
+
+
+case class CreateVariable(name: LogicalPlan, defaultExpr: Expression, replace: Boolean) extends UnaryNode with Command {
```

Comment:
```suggestion
case class CreateVariable(name: String, defaultExpr: Expression, replace: Boolean) extends UnaryNode with Command {
```
why name is the `LogicalPlan`? it makes no sense.

Reply:
Fixed